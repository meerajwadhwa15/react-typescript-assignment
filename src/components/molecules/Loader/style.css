.loader {
  margin: 15px;
  text-align: center;
}