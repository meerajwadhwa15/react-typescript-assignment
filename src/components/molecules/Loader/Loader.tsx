import React, { FunctionComponent } from 'react'
import './style.css'

export type Props = {}

const Loader: FunctionComponent<Props> = (props) => (
  <div className="loader">Loading...</div>
)

export default Loader
