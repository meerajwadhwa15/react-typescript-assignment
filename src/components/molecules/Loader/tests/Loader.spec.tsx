import React from 'react'
import { shallow } from 'enzyme'
import { create } from 'react-test-renderer'
import Loader, { Props } from '../Loader'

const props: Props = {}

describe('Loader Enzyme testing', () => {
  test('renders without crashing', () => {
    shallow(<Loader {...props} />)
  })
})

describe('Loader Snapshot testing', () => {
  test('Matches the snapshot', () => {
    const component = create(<Loader {...props} />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
