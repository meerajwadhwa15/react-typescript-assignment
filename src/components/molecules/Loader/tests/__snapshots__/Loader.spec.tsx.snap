// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Snapshot testing Matches the snapshot 1`] = `
<div
  className="loader"
>
  Loading...
</div>
`;
